toolkit/scripts/containerized-build/resources/mariner-3_repo

```diff
@@ -0,0 +1,9 @@
+[mariner-3.0-daily-build]
```

Comment:
I'm curious why you need this?

Reply:
We don't have a repo file for daily-build-id for 3.0. Adding it here, so we can tdnf install from daily-build repo, and disable PMC until we get a PMC for 3.0